tests/tactics.lean

```diff
@@ -19,6 +19,9 @@ by solve_by_elim
 example {α : Type} {a b : α → Prop} (h₀ : ∀ x : α, b x = a x) (y : α) : a y = b y :=
 by solve_by_elim
 
+example {α : Type} {a b : α → Prop} (h₀ : b = a) (y : α) : a y = b y :=
+by solve_by_elim
```

Comment:
In this case ``solve_by_elim { discharger := `[cc] }`` also works.

Reply:
`cc` is a bit heavy. If you can avoid it, it's good to avoid it.